docs/pages/build-reference/app-versions.md

```diff
@@ -0,0 +1,110 @@
+    }
+  }
+}
+```
+
+</Collapsible>
+
+## Remote version source
+
+You can configure your project to rely on EAS servers to store and manage the version of your app, to do that add `{ "cli": { "appVersionSource": "remote" } }` in your **eas.json**. The remote version will be initialized with the value from a local project — if you would like to explicitly set the value directly, or EAS CLI is not able to detect what version an app is on, you can use the `eas build:version:set` command. EAS stores version information scoped by account, slug, platform, and application ID/bundle identifier — so, for example, if you are building variants with different application IDs or bundle identifiers, versioning will be independent for each of them.
+
+If you want to build your project locally in Android Studio or Xcode using the same version stored remotely on EAS, you can update your local project with the remote versions using `eas build:version:sync`.
+
+Enabling the `autoIncrement` option in the remote app version source mode is currently only available for `versionCode`/`buildNumber`. 
+
+When using a remote app version source, the values in **app.json** will not be updated when the version is incremented remotely, and so the local and remote values will fall out of sync. The remote source values will be set on the native project when running a build, and they are the source of truth  — however, the values specified in your **app.json** will be present in `Constants.expoConfig` and `Constants.manifest` exposed by `expo-constants`. Use `expo-application` to determine your application version at runtime instead, and remove `versionCode`/`buildNumber` from your **app.json**.
+
+### Limitations
+
+- `eas build:version:sync` command on Android does not support bare projects with multiple flavors, but the rest of the remote versioning functionality should work with all projects.
+- `autoIncrement` does not support the `version` option.
+- It's not supported if you are using EAS Update and runtime policy set to `"runtimeVersion": { "policy": "nativeVersion" }`. For similar behavior, use the `"appVersion"` policy instead.
+
+### Recommended workflow
+
+The main goal of this feature is to avoid manual changes to the project every time you are uploading a new archive to run it on TestFlight / Play Store testing channels. When you are doing a production release, the user-facing version change should be explicit.
+
+We recommend updating `version` field after a new build goes live in the store, especially if you are using `expo-updates` with an automatic runtime version policy. This marks the beginning of a new development cycle for a new version of your app. [Learn more about deployment patterns](/eas-update/deployment-patterns.md).
+
+## Local version source
+
+By default, the source of truth for a project versions is the local project source code itself. In this case, EAS does not write to the project, it reads the values and builds projects as they are.
```

Comment:
```suggestion
The source of truth for project versions is the local project's source code. In this case, EAS does not write to the project, it reads the values and builds projects as they are.
```

Reply:
```suggestion
By default, the source of truth for project versions is the local project source code itself. In this case, EAS does not write to the project, it reads the values and builds projects as they are.
```
"By default" is essential here, both local and remote values can be a source of truth depending on eas.json config